apps/ledger-live-desktop/src/renderer/families/celo/operationDetails.jsx

```diff
@@ -58,7 +58,7 @@ const OperationDetailsExtra = ({ operation, type, account }: OperationDetailsExt
               <OpDetailsData>
```

Comment:
Scroll down to [libs/ledger-live-common/src/families/celo/api/hubble.ts](https://github.com/LedgerHQ/ledger-live/pull/830/files#diff-71f32da087d2549d3276a66e5e92d9d05a1eb5daf998ef3b74694eb07a2055e4)- the changes in that file are the foundation of all of the other changed files in this PR.